File: api/v2010/media/delete-example-1/delete-example-1.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

@client.messages('MM800f449d0399ed014aae2bcc0cc2f2ec')
       .media('ME557ce644e5ab84fa21cc21112e22c485')
       .delete

File: api/v2010/media/delete-example-1/delete-example-1.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Rest.Api.V2010.Account.Message;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        MediaResource.Delete(
            pathMessageSid: "MM800f449d0399ed014aae2bcc0cc2f2ec",
            pathSid: "ME557ce644e5ab84fa21cc21112e22c485"
        );
    }
}
